### PR TITLE
GCD.c

### LINKED ISSUE #2506: GCD in C
Please assign me this issue

### PR DESCRIPTION
**Pull request by anshuman1510**
Fixes #2506 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
please include the issue number next to `Fixes #`

**Comment by anshuman1510** [comment]
Done

File: Gcd/gcd.c [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
```
[the remark this answers is not in the document]
Comment (anshuman1510) [reply Gcd/gcd.c]: Added a new line after include statement

File: Gcd/gcd.c [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+/* Function for gcd */
+int gcd (int a, int b)
+{
+	if (b==0)
+	return a;
+	/* Using recursion for checking gcd*/
+	else 
+	return gcd (b,a%b);
+}
+main()
```
[the remark this answers is not in the document]
Comment (anshuman1510) [reply Gcd/gcd.c]: Done

File: Gcd/gcd.c [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+/* Function for gcd */
+int gcd (int a, int b)
+{
+	if (b==0)
+	return a;
```
[the remark this answers is not in the document]
Comment (anshuman1510) [reply Gcd/gcd.c]: Done

File: Gcd/gcd.c [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+{
+	if (b==0)
+	return a;
+	/* Using recursion for checking gcd*/
+	else 
+	return gcd (b,a%b);
+}
+main()
+{
+	int a,b,hcf;
```
[the remark this answers is not in the document]
Comment (anshuman1510) [reply Gcd/gcd.c]: Done

File: Gcd/gcd.c [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+/* Function for gcd */
+
+int gcd (int a, int b)
+{        
+    if (b==0)
+        return a;
+     /* Using recursion for checking gcd*/
+    else 
+        return gcd (b,a%b);
```
Comment (asha15) [inline Gcd/gcd.c]: keep space around operators
keep a space after comma

File: Gcd/gcd.c [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+main()
+{      
+    int a, b, hcf;
+    printf ("Input two numbers : ");
+    scanf ("%d%d",&a,&b);
+    /* Function Calling*/
+    hcf = gcd (a,b);
```
Comment (asha15) [inline Gcd/gcd.c]: keep a space after comma

File: Gcd/gcd.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (anshuman1510) [reply Gcd/gcd.c]: Done

File: Gcd/gcd.c (hunk 6 above)
Comment (anshuman1510) [reply Gcd/gcd.c]: Done

File: Gcd/gcd.c [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+     /* Using recursion for checking gcd*/
+    else 
+        return gcd (b, a % b);
+}
+
+main()
+{      
+    int a, b, hcf;
+    printf ("Input two numbers : ");
+    scanf ("%d %d",&a, &b);
```
Comment (asha15) [inline Gcd/gcd.c]: keep a space after comma

**Comment by asha15** [comment]
squash your commit and push it

**Pull request closed without merging**